css/properties.json

```diff
@@ -173,22 +173,22 @@
     "groups": [
       "Microsoft Extensions"
     ],
-    "initial": "false",
+    "initial": "none",
     "appliesto": "nonReplacedBlockAndInlineBlockElements",
     "computed": "asSpecified",
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
   "-ms-filter": {
-    "syntax": "<ms-filter-specification>",
+    "syntax": "<string>+",
```

Comment:
`<string>+` -> `<string>`
According to [MSDN](https://msdn.microsoft.com/en-us/library/ms530752(v=vs.85).aspx), that's a single string with comma separated values inside the string. No multiplier is needed.